src/constants/chainInfo.ts

```diff
@@ -167,4 +169,28 @@ export const CHAIN_INFO: ChainInfoMap = {
     logoUrl: polygonMaticLogo,
     nativeCurrency: { name: 'Polygon Mumbai Matic', symbol: 'mMATIC', decimals: 18 },
   },
+  [SupportedChainId.CELO]: {
+    networkType: NetworkType.L1,
+    blockWaitMsBeforeWarning: ms`10m`,
+    bridge: 'https://www.portalbridge.com/#/transfer',
+    docs: 'https://docs.celo.org/',
+    explorer: 'https://explorer.celo.org/',
+    infoLink: 'https://info.uniswap.org/#/',
+    label: 'Celo',
+    logoUrl: celoLogo,
+    nativeCurrency: { name: 'Celo', symbol: 'CELO', decimals: 18 },
+    defaultListUrl: CELO_LIST,
+  },
+  [SupportedChainId.CELO_ALFAJORES]: {
+    networkType: NetworkType.L1,
+    blockWaitMsBeforeWarning: ms`10m`,
+    bridge: 'https://www.portalbridge.com/#/transfer',
```

Comment:
Does this bridge work for alfajores?

Reply:
no the bridge only works for mainnet. I was following suit with the config for the other testnets (which also don't work with testnets)